notebooks/covid_meds/covid_meds.Rmd

```diff
@@ -185,6 +185,8 @@ df_scmd_names <- df_scmd %>%
            ods_code == "R1E" ~ "QNC", # Staffs
            ods_code == "TAD" ~ "QWO", # W Yorks
            ods_code == "TAJ" ~ "QUA", # Black country
+           ods_code == "TAH" ~ "QF7", # South Yorkshire & Bassetlow
+           ods_code == "TAF" ~ "QMJ", # North Central London
```

Comment:
thanks! where did you get these codes from?

Reply:
I just googled e.g. "NHS Trust TAF" to find the name (though we can also find this in the trust file supplied alongside the scmd file). Normally they have a name like "NHS Sheffield X Trust" and then I check to see what STP other Sheffield Trusts are in.

Reply:
thx!